ecommerce/models.py

```diff
@@ -76,6 +73,19 @@ def to_dict(self):
         data['lines'] = [serialize_model_object(line) for line in self.line_set.all()]
         return data
 
+    def get_klass(self):
+        """
+        klass being paid for
+
+        Returns:
+            Klass: klass that order is for.
+
+        """
+        line = self.line_set.first()
+        if not line:
+            return ""
```

Comment:
I don't think returning a string is the right thing here, `None` would be more appropriate.

Reply:
✅